app/templates/editions/octane.hbs

```diff
@@ -0,0 +1,32 @@
+<h1>The Octane Edition of Ember</h1>
+<p class="intro">
+The <strong>preview</strong> of Ember Octane has begun. This means that everything here is experimental and subject to change. Keep that in mind while you peruse this information! 
+</p>
+<ul class="decorative-boxes">
+  <li><a href="https://emberjs.github.io/rfcs/0364-roadmap-2018.html">Read The Roadmap RFC</a></li>
+  <li><a href="https://deploy-preview-455--ember-guides.netlify.com/release/upgrading/editions/">Read the Upgrade Guide</a></li>
+  <li><a href="https://guides.emberjs.com/release/reference/syntax-conversion-guide/">Read The Syntax Conversion Guide</a></li>
+</ul>
+
+<h2>How you can help</h2>
+<p>During this preview phase, we are asking the community to help us in any way they can. Try Octane out in your app. Report bugs back to us. Let us know where your pain points are. Read the guides and see if they make sense to you. It genuinely takes all parts of our community working together to make this a polished, complete project- and no contribution is too small!</p>
+
+<p><strong>We will be able to declare a stable Octane edition faster with the community's help!</strong></p>
+<p>As in all areas of Ember, there are different ways you can contribute. Visit the #st-octane channel on the Ember Discord server to get started today.</p>
+<ul class="decorative-boxes">
+  <li><a href="https://github.com/emberjs/ember.js/issues/17234">Implement Features</a></li>
+  <li><a href="https://github.com/ember-learn/guides-source/issues/394">Write Guides</a></li>
+  <li><a href="https://github.com/ember-learn/guides-source/issues/588">Review & Edit Guides</a></li>
+</ul>
+
+<h2>Frequently Asked Questions</h2>
+<details>
+  <summary>What Happened to Module Unification?</summary>
+  Ember's conventions for project layout and file naming are central to the developer experience. It's crucial that we get both the technical and ergonomic details right. I wanted to provide an update about Module Unification and our plans for the file structure in Ember Octane.
+  <br/> In short, we do not plan to ship Module Unification in Octane. 
+  <br/>Read the full statement about Module Unification on our blog.
```

Comment:
Probably worth linking to the blog post here?

Reply:
Can't put an interactive element inside of another interactive element. That's why I re-worded the prose here.

Reply:
If we can’t put links inside a `details` element, we should probably use a non-interactive element instead. The inability to use hyperlinks is a big cost and I don’t think hiding the content by default is necessary here. People may want to skim the content without having to click a bunch of disclosure triangles.

Reply:
Sounds good. Will change. 👍

Reply:
The part "I wanted to provide" reads a little weird. I would change that to "We wanted to provide". Because this is not a blog post, there is not a reference to an author in the page.